pyproject.toml

```diff
@@ -101,5 +101,6 @@ exclude = [
     "build",
     "doc",
     "tests",
+    "*/__init__.py",
```

Comment:
The alpenhorn solution here is:
```
[tool.ruff.lint.per-file-ignores]
"alpenhorn/*/__init__.py" = ["I", "F401"]
```

You may be able to do something similar to better fine-tune the disabled tests for `__init__.py` files.